### PR TITLE
Lastline fix

### PR DESCRIPTION
**Pull request by esharf**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues 
fixes: https://github.com/demisto/etc/issues/22295

## Does it break backward compatibility?
   - [ ] Yes
   - [x] No

## Must have
- [x] [CHANLOG](https://github.com/demisto/content/edit/lastline_fix/Integrations/Lastline_v2/CHANGELOG.md?pr=%2Fdemisto%2Fcontent%2Fpull%2F5833)
- [x] [README](https://github.com/demisto/content/edit/lastline_fix/Integrations/Lastline_v2/README.md?pr=%2Fdemisto%2Fcontent%2Fpull%2F5833)

### CODE REVIEW

**Comment by kirbles19** [comment]
@esharf Done.

File: Integrations/Lastline_v2/Lastline_v2.py [hunk 1]
```diff
@@ -114,6 +113,10 @@ def get_status_and_time(self, uuids) -> List[List]:
         return task_list
 
     def http_request(self, path: str, headers=None, files=None) -> Dict:
+        if files:
+            with open(files, 'rb') as _file:
+                files = {'file': (files, _file.read())}
```
Comment (yaakovi) [inline Integrations/Lastline_v2/Lastline_v2.py]: don't override `files` parameter, use a different name.

File: Integrations/Lastline_v2/Lastline_v2.py [hunk 2]
```diff
@@ -113,8 +112,12 @@ def get_status_and_time(self, uuids) -> List[List]:
             task_list.append([uuid, task_time.replace(' ', 'T'), status])
         return task_list
 
-    def http_request(self, path: str, headers=None, files=None) -> Dict:
-        result: Dict = self._http_request('POST', path, params=self.command_params, headers=headers, files=files)
+    def http_request(self, path: str, headers=None, file=None) -> Dict:
+        if file:
```
Comment (Itay4) [inline Integrations/Lastline_v2/Lastline_v2.py]: dont use reserved words like `file`, you can change to something like `file_`

File: Integrations/integration-Lastline_CHANGELOG.md [hunk 3]
```diff
@@ -1,12 +1,6 @@
 ## [Unreleased]
-
-
-## [19.12.0] - 2019-12-10
--  
-
-## [19.9.1] - 2019-09-18
--
-
+  - Deprecate lastline v1
```
Comment (Itay4) [inline Integrations/integration-Lastline_CHANGELOG.md]: @kirbles19 you reviewed this ?

Comment (kirbles19) [reply Integrations/integration-Lastline_CHANGELOG.md]: Deprecated. Use the Lastline v2 integration instead.

**Pull request merged**